ee_core/include/ee_core.h

```diff
@@ -22,9 +22,8 @@
 #include <smod.h>
 
 #ifdef __EESIO_DEBUG
-#include <sio.h>
-#define DPRINTF(args...) sio_printf(args)
-#define DINIT()          sio_init(38400, 0, 0, 0, 0)
+#define DPRINTF(args...) ;
+#define DINIT() ;
```

Comment:
Semicolon here is causing the formatting issue. The format check is more verbose now and actually tells you where the error is:

https://github.com/ps2homebrew/Open-PS2-Loader/actions/runs/3932722527/jobs/6725618641